lib/friendly_shipping/services/ups_freight/generate_email_options_hash.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module FriendlyShipping
+  module Services
+    class UpsFreight
+      class GenerateEmailOptionsHash
+        def self.call(email_options:)
+          {
+            EMailInformation: {
```

Comment:
Wow, I hate how they want this to be "EMail" and not "Email." Looks terrible!

Reply:
CamElCaSe ftw